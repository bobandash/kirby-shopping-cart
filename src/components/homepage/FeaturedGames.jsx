import styles from './FeaturedGames.module.css';
import sharedStyles from './shared.module.css';
import kirbyForgottenLand from '../../assets/kriby-forgotten-land-promo.jpg'

function FeaturedGames(){
  return (
    <section className = {styles["featured-items-container"]}>
      <div className = {sharedStyles.container}>
        <h1 className = {sharedStyles.header}>Featured Games</h1>
        <div className = {styles["game-images-container"]}>
          <button className = {styles["caret-btn"]}><i className="fa-solid fa-angle-left"></i></button>
          <img src = {kirbyForgottenLand} className = {styles["game-image"]} />
          <button className = {styles["caret-btn"]}><i className="fa-solid fa-angle-right"></i></button>
        </div>
        
      </div>
    </section>
  )
}

export default FeaturedGames;