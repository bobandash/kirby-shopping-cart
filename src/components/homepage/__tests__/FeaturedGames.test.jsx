import FeaturedGames from "../FeaturedGames";
import {render} from "@testing-library/react";
import { describe, it, expect } from 'vitest';

describe('Featured Games Component', () => {
  it('renders featured games component', () => {
    const { container } = render(<FeaturedGames />);
    expect(container).toMatchSnapshot();
  })
})