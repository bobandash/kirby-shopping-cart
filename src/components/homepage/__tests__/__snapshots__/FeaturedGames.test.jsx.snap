// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Featured Games Component > renders featured games component 1`] = `
<div>
  <section
    class="_featured-items-container_b6f7f5"
  >
    <div
      class="_container_59ef55"
    >
      <h1
        class="_header_59ef55"
      >
        Featured Games
      </h1>
      <div
        class="_game-images-container_b6f7f5"
      >
        <button
          class="_caret-btn_b6f7f5"
        >
          <i
            class="fa-solid fa-angle-left"
          />
        </button>
        <img
          class="_game-image_b6f7f5"
          src="/src/assets/kriby-forgotten-land-promo.jpg"
        />
        <button
          class="_caret-btn_b6f7f5"
        >
          <i
            class="fa-solid fa-angle-right"
          />
        </button>
      </div>
    </div>
  </section>
</div>
`;
